.gitignore

```diff
@@ -1,11 +1,13 @@
+/archives
 /i386
 /sparc
 
-components/components.mk
-components/depends.mk
+components/*.mk
+components/**/build
```

Comment:
They're both autogenerated content that shouldn't be committed to git and filling git status with them doesn't help anyone who keeps a fairly mature tree around.